src/snyk/common/error/errorReporter.ts

```diff
@@ -78,6 +78,12 @@ export class ErrorReporter {
     }
   }
 
+  static configureScope(requestId: string) {
+    Sentry.configureScope(scope => {
```

Comment:
This seems to be a global level configuration. Local scopes is probably a way to go here in order to operate on a single reported event.

Reply:
ah good shout.